CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 3.10.1 (2022-08-16)
+
+### Fixed
+
+* Fix rails-console (#391)
```

Comment:
I normally use a wording stating the problem being fixed, not the solution, and link to the PR. Let's say something like

```suggestion
* Rails console loading a debugger REPL instead of the standard Pry REPL(#392)
```